pkg/steps/artifacts.go

```diff
@@ -253,7 +254,7 @@ func copyArtifacts(podClient PodClient, into, ns, name, containerName string, pa
 		size += h.Size
 	}
 
-	if size > 0 {
+	if size > 1*1000*1000 {
```

Comment:
Why? Document tpls

Reply:
Reduce pointless output.  Will comment.  The only reason to print it is when we upload lots of content and the job just sits there.